libs/designsystem/page/src/page.component.ts

```diff
@@ -568,6 +584,19 @@ export class PageComponent
     };
   }
 
+  private patchIonLastFocussed() {
+    const ionLastFocus: HTMLElement =
+      this.ionContentElement.nativeElement.querySelector('[ion-last-focus="true"]');
+
+    // When a focusManagerPriority is set, Ionic handles moving focus back to the element that presented the current view.
+    // Unfortunately, we need to schedule a removal of tabindex="-1" from ionLastFocus element after Ionic might have set it
+    // to work around elements becoming inaccessible after navigating back: https://github.com/ionic-team/ionic-framework/issues/29875
+    setTimeout(() => {
```

Comment:
I don't know much about how Ionic works, but I'm curious if this can be achieved within the same task? So instead of using setTimeout we can could use queueMicrotask?

Reply:
Great suggestion, but seems like it needs to happen in a new task, as the tabindex manipulation happens during some `afterTransition` logic in ionic that already is scheduled as its own new task: 
https://github.com/ionic-team/ionic-framework/blob/4317da080ca26a27575a7f7905d2703c89b685cc/core/src/utils/transition/index.ts#L23C5-L23C14